cli/viam/main.go

```diff
@@ -804,21 +801,23 @@ func main() {
 				},
 			},
 			{
-				Name:  "module",
-				Usage: "manage your modules in Viam's registry",
+				Name:            "module",
+				Usage:           "manage your modules in Viam's registry",
+				HideHelpCommand: true,
 				Subcommands: []*cli.Command{
 					{
 						Name:  "create",
 						Usage: "create & register a module on app.viam.com",
 						Description: `Creates a module in app.viam.com to simplify code deployment.
 Ex: 'viam module create --name my-great-module --org-id <my org id>'
-Will create the module and a corresponding meta.json file in the current directory. 
+Will create the module and a corresponding meta.json file in the current directory.
 
-If your org has set a namespace in app.viam.com then your module name will be 'my-namespace:my-great-module' and 
-you wont have to pass a namespace or orgid in future commands. Otherwise there we be no namespace
-and you will have to provide the org id to future cli commands and can't make your module public until you claim one.
+If your org has set a namespace in app.viam.com then your module name will be 'my-namespace:my-great-module' and
+you won't have to pass a namespace or org-id in future commands. Otherwise there will be no namespace
+and you will have to provide the org-id to future cli commands. You cannot make your module public until you claim an org-id.
 
-Next, update your meta.json and use 'viam module update' to push those changes to app.viam.com`,
+After creation, update your meta.json and use 'viam module update' to push changes to app.viam.com`,
```

Comment:
Nit: don't use update to mean anything other than `viam update` when using both in same sentence. Does the first instance of `update` mean something specific, like "edit the meta.json file with your new namespace" or similar? If so, recommend stating that ask explicitly.

If instead it just means "use viam module update to push your new module to app.viam.com" then I would simplify down to just that.

Reply:
Good point; used the second, simplified language.

Reply:
Ah, I think I found a bit more context on this when reviewing the upcoming [CLI-upload-to-registry](https://docs.google.com/document/d/1p9tVQwt_R9AECon8y4LmuVTu6nVDD33C0cA7OxR2gNY/edit) stuff.

It looks like the full module creation process is:

```
## Create template meta.json locally, (and maybe register new 'module-id' name from Viam app?):
viam module create --name 'module-id' --org_id 'abcbacbacbacbacbacbacbac'
## Add our module-specific configuration to this new meta.json file, which otherwise contains no unique values:
vi meta.json
## Update (something!) with the new module-specific configurations we just made. Maybe Viam App, or maybe this is local like git staging:
viam module update --name 'module-id' --org_id 'abcbacbacbacbacbacbacbac'
viam module upload --org_id 'abcbacbacbacbacbacbacbac' --platform "darwin/amd64" --version "1.0.0-dfsafaf" --module meta.json packaged-module.tar.gz
```

If I'm reading the doc right, it looks like:

- `create` will always be accompanied by an `update` then an `upload` (unless ... offline module use??)
- `update` will always be accompanied by an `upload` (unless ... offline module use??)
- `upload` is the command that pushes changes to `app.viam.com`, not `update` (though all three commands return a Viam App URL that includes the `module-id`, suggesting Viam App is made aware of the changes.)

As such, since each will be used so closely alongside the others, I think that each of the three of these needs to make prominent mention of the other two: I'd love to see the descriptions for `update` and `upload` get enhanced alongside `create`.

I'm actually about to write the documentation for this process in [DOCS-950](https://viam.atlassian.net/browse/DOCS-950), so I'd love to match inline help text to docs description!

How about: merge this as is, skipping this comment, and I'll reach out to you afterwards about these three commands specifically once I've documented the process?

Reply:
> How about: merge this as is, skipping this comment, and I'll reach out to you afterwards about these three commands specifically once I've documented the process?

That sounds perfect; thanks @andf-viam. I agree that help messages here should pretty much match docs description. I'll keep this as-is for now.